go-controller/pkg/ovn/master.go

```diff
@@ -107,6 +107,34 @@ func (oc *Controller) SetupMaster(masterNodeName string) error {
 		return err
 	}
 
+	// Determine SCTP support
```

Comment:
@trozet could we put this in a function of it's own? like `oc.SCTPSupport, err = detectSCTPSupport()` or something like that

Reply:
yeah I will do that when i squash/rebase. Will just wait for @girishmg review before I do that.